packages/okta-angular/src/okta/services/okta.service.ts

```diff
@@ -167,7 +167,7 @@ export class OktaAuthService {
      * Returns the referrer path from localStorage or app root.
      */
     getFromUri(): { uri: string, extras: NavigationExtras } {
-      const referrerPath = localStorage.getItem('referrerPath');
+      const referrerPath = localStorage.getItem('referrerPath') || 'false';
```

Comment:
Is it correct that this is `'false'` and not `false` ?

Reply:
Yes, all items in localStorage are strings, see `JSON.parse` on the next line. Now that we have tighter tslint, it is being a stickler about a possibly undefined value being passed to JSON.parse. If we do pass an undefined value it will THROW. So having a fallback of 'false' will allow the subsequent logic to work as intended, it will parse and evaluate to `false`

Reply:
No chance that `'false'` is going to evaluate as truthy somewhere in our code?

Reply:
This is a local variable only used for logic in this function. The value of `referrerPath` is not returned from the function or saved anywhere. I could move the fallback a couple lines forward to here if you prefer:
```
 const path = JSON.parse(referrerPath || 'false')
```

Reply:
Alternatively I can restructure it with some `if / else`

Reply:
I chose the 2nd option